app/controllers/api/v0/api_controller.rb

```diff
@@ -232,15 +232,19 @@ def export_public
     end
   end
 
+  def current_user
+    current_api_user || super
+  end
+
   # Find the user that owns the access token.
   # From: https://github.com/doorkeeper-gem/doorkeeper#authenticated-resource-owner
   private def current_api_user
     @current_api_user ||= User.find_by_id(doorkeeper_token&.resource_owner_id) if doorkeeper_token&.accessible?
   end
 
   private def require_user!
-    raise WcaExceptions::MustLogIn.new if current_api_user.nil? && current_user.nil?
-    current_api_user || current_user
+    raise WcaExceptions::MustLogIn.new if current_user.nil?
+    current_user
```

Comment:
In Ruby, you can implement this pattern as `current_user || raise "Nope."`

Reply:
that raises a syntax error

Reply:
works with brackets though